core/browser_events.js

```diff
@@ -40,24 +42,25 @@ Blockly.browserEvents.Data;
  *     should prevent the default handler.  False by default.  If
  *     opt_noPreventDefault is provided, opt_noCaptureIdentifier must also be
  *     provided.
- * @return {!Blockly.browserEvents.Data} Opaque data that can be passed to
+ * @return {!Data} Opaque data that can be passed to
  *     unbindEvent_.
  * @public
```

Comment:
I think we aren't having `@public` anymore, do we want to remove here?

Reply:
sgtm